src/components/FeatureViews/Onboarding/BusinessInfo.js

```diff
@@ -1,68 +1,222 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import TextField from '@material-ui/core/TextField';
-import Container from '@material-ui/core/Container';
-
 import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
 import { makeStyles } from '@material-ui/core/styles';
+import { OnboardingContext } from './OnboardingContext';
+import { useSessionStorage } from '../../../utils';
+import OnboardingControls from './OnboardingControls';
+import Loading from '../../OmouComponents/Loading';
+import gql from 'graphql-tag';
+import { useQuery, useMutation } from '@apollo/client';
+import Grid from '@material-ui/core/Grid';
+import PropTypes from 'prop-types';
 
 const useStyles = makeStyles(() => ({
     Text: {
-        marginTop: '65px',
+        marginTop: '24px',
     },
     Subtitle: {
         fontFamily: 'Arial, Helvetica Neue, Helvetica, sans-serif',
```

Comment:
does the fontFamily need to be here? I thought it inherits from our design system?

Reply:
yeah, we should be taking this from our font family. Why is this Arial?

Reply:
jk. this was from ryan's code... lmao. we need to fix this